### PR TITLE
#13 Denunciar button is out of panel Servico element in Detalhes do serviço

### PR DESCRIPTION
**Pull request by Yemolai**
This solves the issue #13 
The Denunciar button position was corrected, back into place only with css but it still doesn't work because some Content Security Policy directive infringement in its inline execution, is this CSS correction enough or is it needed to fix the button behaviour too?

image

### CODE REVIEW

**Comment by juuh42dias** [comment]
@Yemolai  do you have any gif to show this behaviour? If you don't that's ok to merge.

**Comment by Yemolai** [comment]
I'll provide it later today, @juuh42dias

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -656,17 +659,18 @@ footer.footer {
...
+	display: flex;
+	align-items: center;
+  	height: 100%;
+  	padding: 0 0.4em;
+    .like-dislike {
+		padding: 0 0.4em;
+		height: 100%;
+		align-self: center;
+		font-size: 1.6em;
+		font-weight: 600;
```
Comment (miglsoares) [inline app/assets/stylesheets/application.scss]: esse espaçamento tá meio esquisito :/ você pode identar esse arquivo editado com dois espaços, pls?

Comment (Yemolai) [reply app/assets/stylesheets/application.scss]: Realmente tá estranho, eu não tinha deixado assim, vou dar uma olhada nisso

**Comment by Yemolai** [comment]
> @Yemolai do you have any gif to show this behaviour? If you don't that's ok to merge.

Produção:
https://www.loom.com/share/c9de3c724ab9483d950e3fc290ac6d11

Dev:
https://www.loom.com/share/905717c048f246fbab713b0b0ce8dc69

This is the behaviour I was talking about

**Pull request merged**